proposals/0000-out-of-npm-artifacts.md

```diff
@@ -0,0 +1,344 @@
+
+## Overview of current artifacts
+
+In order to have an holistic overview of which artifacts we have at the moment and where they live, let’s walk through each of them:
+
+### The React Native Android Archive
+
+Currently we ship a prebuilt version of React Android inside the `react-native` NPM package. The prebuilt is located inside the `android` folder and needs to follow this structure:
+
+```
+// Checksum files have been removed for brevity.
+android
+└── com
+    └── facebook
+        └── react
+            └── react-native
+                ├── 0.70.0
+                │   ├── react-native-0.70.0-debug-sources.jar
+                │   ├── react-native-0.70.0-debug.aar
+                │   ├── react-native-0.70.0-release-sources.jar
+                │   ├── react-native-0.70.0-release.aar
+                │   ├── react-native-0.70.0.module
+                │   └── react-native-0.70.0.pom
+                ├── maven-metadata.xml
+                ├── maven-metadata.xml.md5
+                ├── maven-metadata.xml.sha1
+                ├── maven-metadata.xml.sha256
+                └── maven-metadata.xml.sha512
+```
+
+The `android` folder is effectively a Maven Repository which follows the [official repository layout](https://cwiki.apache.org/confluence/display/MAVEN/Remote+repository+layout). The `.module` file is a [Gradle Module Metadata](https://docs.gradle.org/current/userguide/publishing_gradle_module_metadata.html) file. It allows to publish two variants for debug and release.
+The .aar files contain the Android compiled bytecode for React Native, alongside native libraries (.so).
+
+At this stage, debug symbols for React Native are not distributed (also for the `react-native-0.70.0-debug.aar`) due to constraints in size for the NPM package. Similarly, prefab support is not enabled (i.e. headers are not distributed, libraries can’t depend on native libraries directly and are forced to unzip the .aar manully - See [https://github.com/react-native-community/discussions-and-proposals/issues/423](https://github.com/react-native-community/discussions-and-proposals/issues/423)).
+
+The `-sources.jar` file contains Java/Kotlin sources used by IDEs to offer code visualization support.
+Alongside those files, each file is distribute with its own checksums (.md5, .sha1, .sha256, .sha512).
+GPG Signatures for those files are not distributed (i.e. `.asc` files).
+
+Please note that the files needs to follow the repository layout to properly be consumed.
+
+### The Hermes Android Archive
+
+With [Bundled Hermes](https://reactnative.dev/architecture/bundled-hermes), the prebuilt version of Hermes have been moved inside the `react-native` NPM package. They’re located inside the `android` folder similarly to:[The React Native Android Archive](https://fb.quip.com/vBssAxF1lzFm#temp:C:aHXc98a51248365471ebf15e694f)
```

Comment:
headsup, the link is going through quip so it's unreachable 😅

Reply:
oops Let me find an alternative 👍